services/provider/server/server.go

```diff
@@ -537,6 +543,36 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 			},
 		}),
 	})
+
+	nb := &nbv1.NooBaa{}
+	nb.Name = "noobaa"
+	nb.Namespace = s.namespace
+
+	if err = s.client.Get(ctx, client.ObjectKeyFromObject(nb), nb); err != nil {
+		return nil, fmt.Errorf("failed to get noobaa %v", err)
+	}
+	if nb.Status.Phase != nbv1.SystemPhaseReady {
+		// Noobaa system is not yet ready
+		return extR, nil
+	}
+	nbS3Endpoint := nb.Status.Services.ServiceS3.NodePorts[0]
+	extR = append(extR, &pb.ExternalResource{
+		Name: "S3EndpointProxy",
```

Comment:
@ezio-auditore As per the current console-proxy service declaration, the service name in the consumer cluster must be `s3`:
```
oc -n openshift-storage get consoleplugin odf-console -oyaml
[...]
spec:
  [...]
  proxy:
  [...]
  - alias: s3
    authorization: None
    endpoint:
      service:
        name: s3    <=================================
        namespace: openshift-storage
        port: 443
      type: Service
```

Reply:
@ezio-auditore sorry, in fact we'll be adding a new declaration for the `odf-client-console` plugin, so we can go with your suggested name but with this format:`s3-endpoint-proxy`.

Reply:
@alfonsomthd have updated in the latest commit